### PR TITLE
RDM-9118 FTA organisation adjustment for RDM-8842

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

  [RDM-9118](https://tools.hmcts.net/jira/browse/RDM-9118) _"Increment Assigned User Count FTA organisation adjustment"_
  [RDM-8842](https://tools.hmcts.net/jira/browse/RDM-8842) _"Increment Assigned User Count of Cases when Assigned to Users"_ 

### Change description ###

Increment Assigned User Count FTA organisation adjustment:
* load organisation data from PRD or test stub.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -264,7 +264,7 @@ dependencies {
     }
     compile 'org.jooq:jool-java-8:0.9.14'
 
-    testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.0.0'
+    testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.1.0-RDM-9118.3'
```
Comment (mattnayler) [inline build.gradle:267]: Update version once full release is made including hmcts/befta-fw#84.

File: src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Hemanth.td.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "F-105_User_Hemanth",
+
+  "username": "befta.master.solicitor.benjamin@gmail.com",
+  "password": "[[$CCD_BEFTA_MASTER_SOLICITOR_2_PWD]]"
```
Comment (mattnayler) [inline src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Hemanth.td.json:5]: Confirm username value and the password variable used are correct after @MSancaktutar review of Benjamin & Becky references.

File: src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Olawale.td.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "F-105_User_Olawale",
+
+  "username": "befta.master.solicitor.becky@gmail.com",
+  "password": "[[$CCD_BEFTA_MASTER_SOLICITOR_1_PWD]]"
```
Comment (mattnayler) [inline src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Olawale.td.json:5]: Confirm username value and the password variable used are correct after @MSancaktutar review of Benjamin & Becky references.

File: src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Hemanth.td.json (hunk 2 above)
Comment (MSancaktutar) [reply src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Hemanth.td.json:5]: Yes, correct.

File: src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Olawale.td.json (hunk 3 above)
Comment (MSancaktutar) [reply src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Olawale.td.json:5]: Yes, correct.

**Pull request merged**